rewards-merkle-generator.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+
+WORKDIR=$PWD
+
+printf "${LOG_START}Initializing merkle-distributor submodule...${LOG_END}"
+
+git submodule update --init --recursive --remote --rebase --force
+
+printf "${LOG_START}Installing dependencies...${LOG_END}"
+
+cd "$WORKDIR/include/merkle-distributor"
+yarn
```

Comment:
To avoid adding another package management tool to our stack, we can fork `Uniswap/merkle-distributor` and use https://github.com/imsnif/synp to generate `package-lock.json`. I am not yet sure if it's the best strategy, however, the part in #625 uses `npm`, and this part requires `yarn` so requiring having two package managers to generate rewards end-to-end does not feel right. 

We should go either with `yarn` or with `npm` on all levels and since we still have `npm` everywhere else and we are consistent about it, forking and adjusting to `npm` even as an intermediary step does not feel bad.

Reply:
I think I was able to simplify it without forking and adding intermediary step. Please see https://github.com/keep-network/keep-ecdsa/pull/627/commits/7cbe29f319e3c854a69696c1cef2f5fa6b12071c

Reply:
I still see the benefit of forking the repo in an unlikely case the original one gets deleted. Also, I am a little worried about whether `npm i` results without a lock file will not get us into unpredictable problems one day. Let's

Reply:
Sure, I forked `Uniswap/merkle-distributor` to `keep-network/merkle-distributor` and added dependecy. 

Regarding potential problems with `npm i` I think we greatly reduced the risk by having a local copy of this project. As long as we don't need to modify `package.json` in our local copy, I think it should work just fine. If we want to be extra careful, we can add a `package-lock.json` as well.

Reply:
Let's add `package-lock.json` as well. Based on my understanding of `npm i` and `npm ci`, we are in trouble as long as we do not have `package-lock.json` or as long as `package.json` versions of dependencies are not fixed. The latter is debatable given there is no guarantee transitive dependencies have all versions of their dependencies fixed.

Reply:
https://github.com/keep-network/merkle-distributor/pull/1